qa/integration-tests/src/test/java/io/zeebe/broker/it/incident/IncidentTest.java

```diff
@@ -65,28 +64,35 @@ public void setUp() {
   }
 
   @Test
+  @Ignore("https://github.com/zeebe-io/zeebe/issues/1033")
```

Comment:
Why is this test ignored it still works. I would write it this way:

```
  @Test
  public void shouldCreateAndResolveInputMappingIncident() {
    // given
    deploy();

    workflowClient
        .newCreateInstanceCommand()
        .bpmnProcessId("process")
        .latestVersion()
        .send()
        .join();

    final Record<IncidentRecordValue> incident =
        RecordingExporter.incidentRecords(IncidentIntent.CREATED).getFirst();

    // when
    workflowClient
        .newUpdatePayloadCommand(incident.getValue().getActivityInstanceKey())
        .payload(PAYLOAD)
        .send()
        .join();

    // then
    assertJobCreated("test");
    assertIncidentResolved();
  }
```

Reply:
I would not spend any time on test cases which we are throwing away in the near future, but yeah can fix it.

Reply:
why would we throw this away?